Configuration/Settings.yaml

```diff
@@ -0,0 +1,19 @@
+Neos:
+  Cqrs:
+    EventStore:
+      storage:
+        options:
```

Comment:
That used to be `Neos.EventStore.DatabaseStorageAdapter.persistence`.
Is  `Neos.Cqrs.EventStore.storage.options` a good replacement?